eventlet/support/afinet.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Overall, I think it belongs to greenio/base, probably no need to create another module.